src/styles/utils/variables.scss

```
@@ -0,0 +1,6 @@
+$card-border: #ccc;
+$card-hover-color: #34568b;
+$header-box-shadow: rgba(0, 0, 0, 0.05);
+$nav-item-color: #060b35;
+$nav-link-hover: #00acdc;
+$hover-transition: 0.3s;
```

Variable names should as obvious as possible.
```suggestion
$card-border-color: #ccc;
$card-hover-color: #34568b;
$header-box-shadow: rgba(0, 0, 0, 0.05);
$nav-item-color: #060b35;
$nav-link-hover-color: #00acdc;
$hover-transition-duration: 0.3s;
```